sassc.c

```diff
@@ -202,6 +202,7 @@ void print_usage(char* argv0) {
     printf("   -m, --sourcemap         Emit source map.\n");
     printf("   -M, --omit-map-comment  Omits the source map url comment.\n");
     printf("   -p, --precision         Set the precision for numbers.\n");
+    printf("   -a, --sass              Force the format when using stdin.\n");
```

Comment:
Overall LGTM! Thanks!

Personally would have left out the "file (or stdin)" bits, as it seems a bit "redundant":
as in: `Treat input as indented syntax.`. Not sure it's worth shortening though ...